documentation/source/install/on_premise_solutions.rst

```diff
@@ -310,7 +310,7 @@ Run Sample Applications on the OpenEBS Setup
          0|     3|     0.000|     1.109|      0.000|     10.602|          0|        378|
       name@MayaMaster:~$
 
-.. In case of dedicated installations, the maya volume list and maya volume stats commands can be executed directly on the maya server host console.
+.. In case of dedicated installations, the maya `volume list` and maya `volume stats` commands can be executed directly on the maya server host console.
```

Comment:
1. Add maya also inside the shell coats in both places( `maya volume list`) and 
2. Remove the added architecture-overview-d.png file ( why it is added ?)